SymphonyServerAddon.lua

```diff
@@ -237,11 +253,20 @@ function ParseWebServiceResult(result)
         record = {
             Barcode = data[1],
             Location = data[2],
-            CallNumber = data[3]
+            CallNumber = data[3],
+            ShelfLocation = nil;

```

Comment:
2 things about this line:

1. You can't actually store a `nil` in a Lua table, so specifying a `nil` key in a Lua table doesn't have any function.
2. There's a misplaced semicolon.

Both of these are fine and should not cause any issues, just figured I should point it out.

Reply:
```lua
data = { 'Some Barcode', 'Some Location', 'Some CallNumber' }

-- ...

record = {
    Barcode = data[1],
    Location = data[2],
    CallNumber = data[3],
    ShelfLocation = nil;
};

for k,v in pairs(record) do
    print(' -> ' .. k)
end

-- -> Barcode
-- -> Location
-- -> CallNumber

record.NewKey = 'Shelf Location'
for k,v in pairs(record) do
    print(' -> ' .. k)
end

-- -> Barcode
-- -> Location
-- -> CallNumber
-- -> NewKey

```